src/app/shared/modal/modal-login/modal-login.component.ts

```diff
@@ -2,7 +2,7 @@ import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
 import { Router } from '@angular/router';
 
 import { AuthenticationService } from '../../services/authentication.service';
-import { User } from '../../interfaces/user.interface.';
+import { CourseService } from '../../services/course.service';
```

Comment:
Try to avoid the long paths like **../../**

Reply:
done, with path mapping in `tsconfig.json `